packages/evm/src/evm.ts

```diff
@@ -295,9 +302,10 @@ export class EVM implements EVMInterface {
       }
     }
 
-    // Safeguard if "process" is not available (browser)
-    if (typeof process?.env.DEBUG !== 'undefined') {
-      this.DEBUG = true
```

Comment:
The env.DEBUG check shouldn't be removed here, instead both should now count in:

```typescript
if (typeof process?.env.DEBUG !== 'undefined' && !opts.deactivateCLDebug) {
  this.DEBUG = true
}
```